lib/globus/client.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require_relative "client/version"
```

Comment:
Should this now be `require "globus/client/version"` like the other require statements below?

Reply:
Actually, `globus/client/version` breaks the test. I think the include is correct (or, `require_relative` makes it "correct") since this file lives at `globus/` so `version.rb` is at `client/version` relative to this file.